trade-finance-manager-api/src/graphql/reducers/mappings/gef-facilities/mapGefFacilityProvidedOn.js

```diff
@@ -1,7 +1,14 @@
-const { titleCaseString } = require('../../../../utils/string');
+const CONSTANTS = require('../../../../constants');
 
 const mapGefFacilityProvidedOn = (details) =>
-  details.map((d) => titleCaseString(d));
+  details.map((basis) => {
```

Comment:
Are you sure you want map here and not reduce? Will keep array same length just empty. Might cause unexptected results?

Reply:
Good shout, i've updated to use reduce